src/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -570,7 +571,14 @@ private async Task<ActionResult> AuthenticateIdPortenToken(string originalToken)
                 string authLevel = token.Claims.Where(c => c.Type.Equals(AuthLevelClaimName)).Select(c => c.Value).FirstOrDefault();
                 string authMethod = token.Claims.Where(c => c.Type.Equals(AuthMethodClaimName)).Select(c => c.Value).FirstOrDefault();
                 string externalSessionId = token.Claims.Where(c => c.Type.Equals(ExternalSessionIdClaimName)).Select(c => c.Value).FirstOrDefault();
+                string scope = token.Claims.Where(c => c.Type.Equals(AuthzConstants.CLAIM_MASKINPORTEN_SCOPE)).Select(c => c.Value).FirstOrDefault();
```

Comment:
Constant is not defined (or included in PR)

Reply:
But it is a old one

Reply:
Moved to other constant file. Not sure why it did not catch it during build. IT WORKED ON MY MACHINE